
Include 3rdparty directory into the main build-system.

Original author is unknown.
(Was either Juan Quintela or Jeff Garzik)

Signed-off-by: Luiz Fernando N. Capitulino <lcapitulino@mandriva.com.br>
Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
[Simplified for Mageia 7 / kernel 5.1.6 and up / tmb]
Signed-off-by: Thomas Backlund <tmb@mageia.org>

---
 Makefile           |    2 +-
 arch/arm/Kconfig   |    2 ++
 arch/arm64/Kconfig |    2 ++
 arch/x86/Kconfig   |    2 ++
 4 files changed, 7 insertions(+), 1 deletion(-)

--- linux-5.8/arch/arm64/Kconfig.orig	2020-08-05 16:08:02.466740952 +0200
+++ linux-5.8/arch/arm64/Kconfig	2020-08-05 16:08:13.368741815 +0200
@@ -1910,3 +1910,5 @@ source "arch/arm64/kvm/Kconfig"
 if CRYPTO
 source "arch/arm64/crypto/Kconfig"
 endif
+
+source "3rdparty/Kconfig"
--- linux-5.8/Makefile.orig	2020-08-05 16:09:32.265748061 +0200
+++ linux-5.8/Makefile	2020-08-05 16:10:02.553750458 +0200
@@ -651,7 +651,7 @@ endif
 ifeq ($(KBUILD_EXTMOD),)
 # Objects we will link into vmlinux / subdirs we need to visit
 core-y		:= init/ usr/
-drivers-y	:= drivers/ sound/
+drivers-y	:= drivers/ sound/ 3rdparty/
 drivers-$(CONFIG_SAMPLES) += samples/
 drivers-y	+= net/ virt/
 libs-y		:= lib/
--- linux-5.8/arch/x86/Kconfig.orig	2020-08-05 16:10:48.805754119 +0200
+++ linux-5.8/arch/x86/Kconfig	2020-08-05 16:11:04.602755370 +0200
@@ -2913,3 +2913,5 @@ source "drivers/firmware/Kconfig"
 source "arch/x86/kvm/Kconfig"
 
 source "arch/x86/Kconfig.assembler"
+
+source "3rdparty/Kconfig"
--- linux-5.8/arch/arm/Kconfig.orig	2020-08-05 16:12:08.314760413 +0200
+++ linux-5.8/arch/arm/Kconfig	2020-08-05 16:12:22.786761559 +0200
@@ -2064,3 +2064,5 @@ source "arch/arm/crypto/Kconfig"
 endif
 
 source "arch/arm/Kconfig.assembler"
+
+source "3rdparty/Kconfig"
